opds.py

```diff
@@ -1179,9 +1179,26 @@ def license_tags(cls, license_pool, loan, hold):
             return tags
 
 
-        holds_kw = dict(total=str(license_pool.patrons_in_hold_queue or 0))
-        if hold and hold.position:
-            holds_kw['position'] = str(hold.position)
+        holds_kw = dict()
+        total = ((license_pool.patrons_in_hold_queue or 0) + 
+                 (license_pool.licenses_reserved or 0))
```

Comment:
I agree that reserved licenses should count towards the total, but I thought that was already included in `patrons_in_hold_queue`. That's how I did it for ODL holds. Patrons with reserved licenses are at the beginning of the queue and have holds with position `0`.

Reply:
You're right, I remember this now.